src/main/java/com/backbase/ct/bbfuel/configurator/ProductSummaryConfigurator.java

```diff
@@ -25,6 +28,8 @@
 @RequiredArgsConstructor
 public class ProductSummaryConfigurator {
 
+    private static final List<String> ARRANGEMENT_STATES = Arrays.asList("Active", "Closed", "Inactive");
```

Comment:
better move this to the generator, and preferably don't duplicate as it can be retrieved from the capability (`arrangementsIntegrationRestClient.getArrangementStates().getStates().stream().map(State::getExternalStateId).collect(Collectors.toList());`)